### PR TITLE
Add virustotal results to version screen

### PR DESCRIPTION
**Pull request by knightzac19**
This adds the frontend portions of https://github.com/satisfactorymodding/smr-api/pull/92 so that users can see the results of virustotal scans.

### CODE REVIEW

**Comment by Vilsol** [comment]
Please make the table design match that of all others
image

**Comment by Vilsol** [comment]
Also please rebase on staging

**Comment by knightzac19** [comment]
> Please make the table design match that of all others image

@Vilsol So I used `thead` instead of the `tbody` for the header, which is why it looks different, I can switch the other two tables to `thead` so they look the same?

File: src/lib/components/versions/VirustotalResults.svelte [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  import type { VirustotalResult } from '$lib/generated';
+  import { getTranslate } from '@tolgee/svelte';
+
+  export let results!: Array<VirustotalResult>;
+  export const { t } = getTranslate();
+</script>
+
+<div class="grid grid-flow-row">
+  <h3 class="my-4 text-2xl font-bold">VirusTotal Results</h3>
+  <table aria-label="Required Mod Dependencies" class="max-w-auto table table-hover !overflow-visible">
```
Comment (Th3Fanbus) [inline src/lib/components/versions/VirustotalResults.svelte:11]: Please update `aria-label`

**Comment by budak7273** [comment]
> > Please make the table design match that of all others image
> 
> @Vilsol So I used `thead` instead of the `tbody` for the header, which is why it looks different, I can switch the other two tables to `thead` so they look the same?

As discussed on the discord, let's go with `tbody` for the header

**Comment by knightzac19** [comment]
@budak7273 Okay I've synced with staging, fixed a small bug I found with the table, added translatable text and moved to tbody.

**Review by Vilsol — APPROVED** [review]

**Pull request merged**